app/controllers/locationOfGoods/QualifierOfIdentificationController.scala

```diff
@@ -45,26 +48,35 @@ class QualifierOfIdentificationController @Inject() (
 
   private val form = formProvider[QualifierOfIdentification]("locationOfGoods.qualifierOfIdentification")
 
-  def onPageLoad(mrn: MovementReferenceNumber, mode: Mode): Action[AnyContent] = actions.requireData(mrn) {
-    implicit request =>
-      val preparedForm = request.userAnswers.get(QualifierOfIdentificationPage) match {
-        case None        => form
-        case Some(value) => form.fill(value)
-      }
+  def onPageLoad(mrn: MovementReferenceNumber, mode: Mode): Action[AnyContent] = actions
+    .requireData(mrn)
+    .andThen(getMandatoryPage(IsSimplifiedProcedurePage)) {
+      implicit request =>
+        val qualifierOfIdentificationValues =
+          if (request.arg == Simplified) QualifierOfIdentification.values else QualifierOfIdentification.normalProcedureValues
```

Comment:
Possible refactor but we have a RadioModelU trait for this exact purpose

Reply:
I don't believe we have that trait in the service at the moment.